packages/account/src/Components/poa-address-mismatch-hint-box/__test__/poa-address-mismatch-hint-box.spec.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import POAAddressMismatchHintBox from '../poa-address-mismatch-hint-box';
+
+describe('POAAddressMismatchHintBox', () => {
+    it('should render POAAddressMismatchHintBox', () => {
+        render(<POAAddressMismatchHintBox />);
+        expect(
+            screen.getByText(
+                /`it appears that the address in your document doesn’t match the address in your deriv profile\. please update your personal details now with the correct address\.`/i
```

Comment:
Why are you using lower-case for all the characters?

Reply:
It's matching using a regex and comparing with lower characters alongside with added escape characters 
Selected using this extension: https://chrome.google.com/webstore/detail/testing-playground/hejbmebodbijjdhflfknehhcgaklhano

Reply:
Thanks Suisin for explanation 😄